doc/whats-new.rst

```diff
@@ -85,6 +85,9 @@ New Features
   invoked. (:issue:`3378`, :pull:`3446`, :pull:`3515`)
   By `Deepak Cherian <https://github.com/dcherian>`_ and
   `Guido Imperiale <https://github.com/crusaderky>`_.
+- Added the :py:meth:`count` reduction method to both :py:class:`DatasetCoarsen`
+  and :py:class:`DataArrayCoarsen` objects. (:pull:`3500`)
+  By `Deepak Cherian <https://github.com/dcherian/>`_
```

Comment:
now that #3516 is merged, this should be
```suggestion
  By `Deepak Cherian <https://github.com/dcherian>`_
```

Reply:
thanks!